storage/namespace.go

```diff
@@ -435,6 +438,8 @@ func (n *dbNamespace) Tick(c context.Cancellable) {
 	n.metrics.tick.madeUnwiredBlocks.Inc(int64(r.madeUnwiredBlocks))
 	n.metrics.tick.mergedOutOfOrderBlocks.Inc(int64(r.mergedOutOfOrderBlocks))
 	n.metrics.tick.errors.Inc(int64(r.errors))
+
+	return multiErr.FinalError()
```

Comment:
I don't think we want to emit stats or set the last tick stats unless err == nil.

Reply:
fixed up